client/lib/wpcom-undocumented/lib/account-recovery-reset.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * `AccountRecoveryReset` constructor.
+ *
+ * @constructor
+ * @param {Object} userData  An object containing either user or firstname, lastname and url
+ * @param {Object} wpcom     WPCOM object
+ * @public
+ */
+function AccountRecoveryReset( userData, wpcom ) {
+	if ( ! ( this instanceof AccountRecoveryReset ) ) {
+		return new AccountRecoveryReset( userData, wpcom );
+	}
+
+	this._userData = userData;
+	this.wpcom = wpcom;
+}
+
+AccountRecoveryReset.prototype.getResetOptions = function() {
+	return this.wpcom.req.get( {
+		body: this._userData,
+		apiNamespace: 'wpcom/v2',
+		path: '/account-recovery/lookup',
+	} );
```

Comment:
> does this turn body into query parameters? might we actually need to issue a POST request instead?

@dmsnell, It does turn `body` into query parameters and I think its fine doing the `GET` request since its not updating the users account in anyway.